client/src/view/component/ResetSessionDialog.tsx

```diff
@@ -0,0 +1,41 @@
+import { Grid, Box, Typography } from "@mui/material";
+import { ResetSession } from "../container/session-reset/ResetSession";
+import { ViewSession } from "../../controller/model/ViewSession";
+import { ResumeButton } from "./ResumeButton";
+
+interface ResetSessionDialogProps {
+  session: ViewSession;
+  setSession: (session: ViewSession | undefined) => void;
+  closeModal: () => void;
+}
+
+export const ResetSessionDialog = ({
```

Comment:
Viskas identiškai kaip DeleteSessionDialog, apart to kad surenderina skirtingą tekstą ir Reset arba Delete mygtuką. Sakyčiau reiktų parefactorinti į 'SessionDialog' komponentą, kuris priima text ir 'OK' (of type JSX.Element) buttoną